docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,162 @@
+# Contributing to Poke-guesser Bot
+
+First off, thanks for taking the time to contribute! ❤️
+
+All types of contributions are encouraged and valued. See the [Table of Contents](#table-of-contents) for different ways to help and details about how this project handles them. Please make sure to read the relevant section before making your contribution. It will make it a lot easier for us maintainers and smooth out the experience for all involved. The community looks forward to your contributions. 🎉
+
+> And if you like the project, but just don't have time to contribute, that's fine. There are other easy ways to support the project and show your appreciation, which we would also be very happy about:
+> - Star the project
+> - Tweet about it
+> - Refer this project in your project's readme
+> - Mention the project at local meetups and tell your friends/colleagues
+> - Join our [discord](#community)
+
+## Table of Contents
+
+- [Contributing to Poke-guesser Bot](#contributing-to-poke-guesser-bot)
+  - [Table of Contents](#table-of-contents)
+  - [Community](#community)
+  - [I Have a Question](#i-have-a-question)
+  - [I Want To Contribute](#i-want-to-contribute)
+    - [Reporting Bugs](#reporting-bugs)
+      - [Before Submitting a Bug Report](#before-submitting-a-bug-report)
+      - [How Do I Submit a Good Bug Report?](#how-do-i-submit-a-good-bug-report)
+    - [Suggesting Enhancements](#suggesting-enhancements)
+      - [Before Submitting an Enhancement](#before-submitting-an-enhancement)
+      - [How Do I Submit a Good Enhancement Suggestion?](#how-do-i-submit-a-good-enhancement-suggestion)
+    - [Your First Code Contribution](#your-first-code-contribution)
+    - [Improving The Documentation](#improving-the-documentation)
+  - [Styleguides](#styleguides)
+    - [Commit Messages](#commit-messages)
+  - [Attribution](#attribution)
+
+
+## Community
+
+If you are interested in this project, please consider joining our [discord](https://discord.gg/WmD5CW3aFu). 
+
+
+## I Have a Question
+
+> If you want to ask a question, we assume that you have read the available [Wiki](https://github.com/GeorgeCiesinski/poke-guesser-bot/wiki).
+
+Before you ask a question, it is best to search for existing [Issues](https://github.com/GeorgeCiesinski/poke-guesser-bot/issues) that might help you. In case you have found a suitable issue and still need clarification, you can write your question in this issue. It is also advisable to search the internet for answers first.
+
+If you then still feel the need to ask a question and need clarification, we recommend the following:
+
+- Open an [Issue](https://github.com/GeorgeCiesinski/poke-guesser-bot/issues/new).
+- Provide as much context as you can about what you're running into.
+- Provide project and platform versions (nodejs, npm, etc), depending on what seems relevant.
+- Label the issue `question`.
```

Comment:
Afaik you can not label PR's yourself if you are neither Owner, Member or Collaborator, but I could be wrong. Not sure if we have a template where we could I think set it to automatically assign a label on creation.

Reply:
Good point. I removed that point from the document.